docs/api.rst

```diff
@@ -195,7 +195,6 @@ Circuits, Operations, and Moments.
     InsertStrategy
     Moment
     moment_by_moment_schedule
-    op_gate_of_type
```

Comment:
Do we normally remove deprecated functions from the API?

Reply:
I don't know the policy here. @Strilanc?

Reply:
I don't know the policy here. @Strilanc?